plane/source/docs/guide-tailsitter.rst

```diff
@@ -71,43 +95,39 @@ Motors are controlled by the Throttle, Throttle Left, Throttle Right outputs:
+The :ref:`AHRS_ORIENTATION<AHRS_ORIENTATION>`, the accelerometer calibration and Level trim
 should all be done for fixed wing flight. Fixed wing flight is
 considered "normal" orientation for a tailsitter.
 
+- when in VTOL flight the nose of the aircraft will try to point vertically for
+  "level" flight
+- when in fixed wing flight the nose of the aircraft will try to point
+  horizontally for "level" flight
+
 Key parameters
 ==============
 
-The key differences between fixed wing flight and hovering for a
+The key differences between fixed wing flight and VTOL for a
 tailsitter are:
 
-- when hovering the copter PID gains will be used (the ones starting
-  with Q_A_RAT_*)
+- when in VTOL the copter stle PID gains will be used (the ones starting
+  with ``Q_A_RAT_x``)
 - when in fixed wing flight the fixed wing PID gains will be used (the
-  PTCH2SRV_* and RLL2SRV_* gains)
-- when hovering the nose of the aircraft will try to point up for
-  "level" flight
-- when in fixed wing flight the nose of the aircraft will try to point
-  forward for "level" flight
-  
-:ref:`Q_TAILSIT_RLL_MX<Q_TAILSIT_RLL_MX>` allows the roll limit angle limit to be set differently from
-:ref:`Q_ANGLE_MAX<Q_ANGLE_MAX>`. If left at zero both pitch and roll are limited by :ref:`Q_ANGLE_MAX<Q_ANGLE_MAX>`. If :ref:`Q_TAILSIT_RLL_MX<Q_TAILSIT_RLL_MX>` is nonzero roll angle will be limited and pitch max angle will still be :ref:`Q_ANGLE_MAX<Q_ANGLE_MAX>`.
-This should be set if your tailsitter can achieve much larger pitch angle than 
-would be safe for roll (some airframes can't recover from high-speed knife-edge flight).
-
-:ref:`Q_TRIM_PITCH<Q_TRIM_PITCH>` can be used to account for any offset in hover pitch angle due to the center of gravity being closer to the belly or top of the fuselage. Set this such that the the vehicle does not drift forwards or backwards in QSTABILIZE or QHOVER with no pilot inputs.
+  ``RLL_RATE_x`` and ``PTCH_RATE_x`` gains)
+- :ref:`Q_TAILSIT_RLL_MX<Q_TAILSIT_RLL_MX>` allows the roll limit angle limit to be set differently from :ref:`Q_ANGLE_MAX<Q_ANGLE_MAX>`. If left at zero, both pitch and roll are limited by :ref:`Q_ANGLE_MAX<Q_ANGLE_MAX>`. If :ref:`Q_TAILSIT_RLL_MX<Q_TAILSIT_RLL_MX>` is nonzero roll angle will be limited and pitch max angle will still be :ref:`Q_ANGLE_MAX<Q_ANGLE_MAX>`. This should be set if your tailsitter can achieve much larger pitch angle than would be safe for roll (some airframes can't recover from high-speed knife-edge flight).
+- :ref:`Q_TRIM_PITCH<Q_TRIM_PITCH>` can be used to account for any offset in hovering pitch angle due to the center of gravity being closer to the belly or top of the fuselage, or to counter fixed wing AHRS trim used to set angle of attack in cruise, instead of using :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>`. Set this such that the the vehicle does not drift forwards or backwards in QSTABILIZE or QHOVER with no pilot inputs.
```

Comment:
should this be "thrust line not through the CG"?

Reply:
done